scripts/release-create-oclif.js

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env node
```

Comment:
Why remove this?

Reply:
Lint said it was unnecessary. The other file in scripts does not have it.

Reply:
I don't know how this script is used but I know it is required for unix systems to run it if they call it directly (e.g. `$ ./scripts/release-create-oclif.js` as opposed to `$ node ./scripts/release-create-oclif.js`).

If you can say for certain it's not going to break an existing process then I'm fine removing it but if not we should leave it in.

Reply:
I'll put it back.